samples/scripts/PowerShell/GenerateOpenApiYaml.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+.SYNOPSIS 
+Generates the OpenApi doc for the specified version and compares it with the baseline to make sure no breaking changes are introduced
+.Parameter SwaggerDir
+The working directory
+.PARAMETER AssemblyDir
+Path for the web projects dll
+.PARAMETER Versions
+Api versions to generate the OpenApiDoc for and compare with baseline
+.PARAMETER SwashbuckleCLIVersion
+Version of SwashbuckleCLI to use
+#>
+
+param(
+    [string]$SwaggerDir,
+
+    [string]$AssemblyDir,
+
+    [String[]]$Versions,
+
+    [string]$SwashbuckleCLIVersion = '6.1.4'
+)
+foreach ($Version in $Versions)
+{
+    write-host "Generating Yaml file for $Version"
+
+    dotnet new tool-manifest --force
+    dotnet tool install --version $SwashbuckleCLIVersion Swashbuckle.AspNetCore.Cli
+    dotnet swagger tofile --yaml --output "$SwaggerDir/$Version.yaml" "$AssemblyDir" $Version
```

Comment:
nit: I think there's an extra slash here, at least based on the pipeline stdout. If you want to, you can use `Join-Path` to properly join these together